2025.06.0 RN hotfix (#5526)
okta/okta-developer-docs: packages/@okta/vuepress-site/docs/release-notes/2025-okta-identity-engine/index.md

The Single Logout (SLO) for IdPs feature boosts security for organizations using shared devices and external IdPs by automatically ending IdP sessions when a user signs out of any app. This feature also requires a fresh authentication for every new user, eliminating session hijacking risks on shared devices. SLO for IdP supports both SAML 2.0 and OIDC IdP connections, which provides robust session management for shared workstations in any environment. See [Configure Single Logout for IdPs](/docs/guides/single-logout/openidconnectidp/main/). <!-- SLO_SUPPORT_FOR_EXTERNAL_IDP (OKTA-946177) -->

#### Expanded use of user.getGroups() function in Okta Expression Language is GA in Preview

> The `user.getGroups()` function can now be used across all features that support Expression Language. See [Group functions](/docs/reference/okta-expression-language/#group-functions) for more information.

Reviewer: ```suggestion
You can now use the `user.getGroups()` function across all features that support Expression Language. See [Group functions](/docs/reference/okta-expression-language/#group-functions).
```

> The `user.getGroups()` function can now be used across all features that support Expression Language. See [Group functions](/docs/reference/okta-expression-language/#group-functions) for more information. <!-- ENABLE_GET_GROUPS_FUNCTION_ELV2 OKTA-945229-->

Author: ```suggestion
You can now use the `user.getGroups()` function across all features that support Expression Language. See [Group functions](/docs/reference/okta-expression-language/#group-functions). <!-- ENABLE_GET_GROUPS_FUNCTION_ELV2 OKTA-945229-->
```
